staging/src/k8s.io/cli-runtime/pkg/resource/query_param_verifier_v3.go

```diff
@@ -106,9 +113,6 @@ func hasGVKExtensionV3(extensions spec.Extensions, gvk schema.GroupVersionKind)
 // the PATCH end-point. Returns true if the query param is supported by the
 // spec for the passed GVK; false otherwise.
 func supportsQueryParamV3(doc *spec3.OpenAPI, gvk schema.GroupVersionKind, queryParam VerifiableQueryParam) bool {
-	if doc == nil || doc.Paths == nil {
```

Comment:
Sorry for mentioning this only now, but shouldn't `supportsQueryParamV3` return an error which either would be nil, `NewPAramUnsupportedError` or another type of error? Then we would be able to check `gvSpec` and `gvSpec.Paths` where they are used? As far as I can see that should be a fairly simple change.

Reply:
I believe I have addressed this. Please let me know what you think.

Reply:
Yeah, thanks, that looks good